teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -123,20 +231,52 @@ interface BuildConfiguration {
     /**
      * Web UI URL for user, especially useful for error and log messages
      */
-    fun getWebUrl(branch: String? = null): String
+    fun getHomeUrl(branch: String? = null): String
 
-    fun fetchBuildTags(): List<String>
+    val buildTags: List<String>
+    val finishBuildTriggers: List<FinishBuildTrigger>
+    val artifactDependencies: List<ArtifactDependency>
 
-    fun fetchFinishBuildTriggers(): List<FinishBuildTrigger>
+    fun setParameter(name: String, value: String)
 
+    fun runBuild(parameters: Map<String, String>? = null,
+                 queueAtTop: Boolean? = null,
```

Comment:
What does `null` means for these boolean parameters?

Reply:
It means default value on TeamCity. But in this case it's probably better to be specific